### ISSUE TITLE
[Feature] Ability to track Work Assignment ID 

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#8314]
As a project manager, I need the ability to track my project's Work Assignment ID.

### Functional requirements

Add an editable text field to the Project Summary page labeled **Work Assignment ID**. 

This column can be added to the`moped_project` table as `work_assignment_id`. 

### Business context

For indefinite delivery/indefinite quantity (IDIQ) contracts, each assignment issued to contractor has a number (often sequential), and each purchase order (DO) will have one **work assignment ID** (WA ID). 

Typically a contractor uses WA ID on invoices to identify the project (eg `WA15_NOV22`).  Often this WA ID shows in the description line in eCapris (within a contract), but that depends a bit how the project was setup in eCapris.

TLDR, a work assignment ID is part of the language of the IDIQ contracts. One purchase order (DO) per work assignment.

![Summary Page – 6 0 - w- Task orders](https://user-images.githubusercontent.com/1463708/151628866-14f6396a-5d4b-45c0-81bf-3339ea0b3396.png)

**Comment by amenity** [comment]
@chiaberry -  please resize as needed. 🙏
